### PR TITLE
0017: Merge Katello documentation to Foreman documentation

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: text/0016-katello-as-a-plugin/0017-merge-katello-docs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[motivation]: #motivation
+
+The separation of documentation serves to push users into feeling there is a difference between Foreman and Katello. And while there is some technical differences currently, the goal is to bring everything in to alignment. One way to do this is by having a central place to go for documentation on all things Foreman and plugins. The expectation is that users will go to theforeman.org and be able to find Katello plugin documentation as you would any other plugins.
+
+# Detailed design
+[design]: #detailed-design
+
+Here are the set of action items for achieving this:
+
+ * Set a redirect from katello.org to theforeman.org Katello plugin page
```
Comment (stbenjam) [inline text/0016-katello-as-a-plugin/0017-merge-katello-docs.md:21]: Can we have redirects per URL with jekyll? That would make the transition smoother. Hate having a deep link redirected to the main page of a new site.

File: text/0016-katello-as-a-plugin/0017-merge-katello-docs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * Carry forward Katello 2.4+ documentation versions
+ * Add a Troubleshooting guide to theforeman.org and include info from katello.org
```
Comment (stbenjam) [inline text/0016-katello-as-a-plugin/0017-merge-katello-docs.md:23]: +1

I think a top-level troubleshooting link from https://theforeman.org/documentation.html would be helpful.  It's kind of spread out now.

A few plugins have their own Troubleshooting guides, foreman has this page: https://theforeman.org/manuals/1.12/index.html#7.Troubleshooting

But most things tend to go here:
  http://projects.theforeman.org/projects/foreman/wiki/Troubleshooting

and here:
  http://projects.theforeman.org/projects/foreman/wiki/ErrorCodes

File: text/0016-katello-as-a-plugin/0017-merge-katello-docs.md (hunk 1 above)
Comment (ehelms) [reply text/0016-katello-as-a-plugin/0017-merge-katello-docs.md:21]: I don't know but we can look in to it.

**Comment by ehelms** [comment]
Efforts related to this have been underway, the biggest of which is to move the full user documentation over and can be reviewed at: https://github.com/theforeman/theforeman.org/pull/726

**Comment by ehelms** [comment]
This effort has been completed and this RFC can be merged.

**Comment by GregSutcliffe** [comment]
Merged, great work everyone!

**Pull request merged**